packages/ui/src/common/password-field.tsx

```diff
@@ -69,7 +69,7 @@ export function PasswordField({
         </Common.FieldState>
         {props.validatePassword ? (
           <Common.FieldState>
-            {({ state, codes, config }) => {
+            {({ state, codes }) => {
```

Comment:
Calling these `validations`, or `failedValidations`, instead of `codes` seems pretty reasonable.